### ISSUE TITLE
Update images to include namespace prefix of specification

### ISSUE THREAD

**Issue by martinweismann** [3MFConsortium/spec_materials#7]
+ add sentence at the end of document convention along those lines:
In this extension specificaion, the prefix "b" maps to the xml-namespace "..." (see Appendix C) ... )

A visualization schema can be found in https://github.com/3MFConsortium/spec_conventions

**Comment by jordig100** [comment]
Martin. Not clear enough where you want to put the note clarifying the convention, Do you mind updating it yourself in the "1.2.1-updates" branch?

**Comment by jordig100** [comment]
Yes, the images were updated in the 1.2.1-updates branch showing the "m:" namespace. I haven't put any paragraph tough explaining that the "m:" is just an example.
For example we could add in the "Document comventions", after the link to the core spec, the following paragraph:

_In this extension specificaion, the prefix "m", used in the diagrams as an example, maps to the xml-namespace "http://schemas.microsoft.com/3dmanufacturing/material/2015/02" (see Appendix E.3 Namespaces)_

**Comment by jordig100** [comment]
Added the following comment in the document conventions:

In this extension specification, as an example, the prefix "m" maps to the xml-namespace "http://schemas.microsoft.com/3dmanufacturing/material/2015/02". See Appendix E.3 Namespaces.
